sources/packages/web/src/components/common/students/StudentFileUploads.vue

```diff
@@ -43,7 +43,7 @@
           >
           <Column field="updatedAt" header="Date Submitted"
             ><template #body="slotProps">{{
-              dateOnlyLongString(slotProps.data.updatedAt)
+              getISODateHourMinuteString(slotProps.data.createdAt)
```

Comment:
Is `createdAt` sent by API? looks like API is sending only updatedAt

Reply:
Yes, you're right. I added the `createdAt` now. Interesting that it didn't failed when I did the change to `slotProps.data.createdAt`. I also changed `<Column field="createdAt"...` now too. It's getting the right field now. Thanks!